Gemfile.lock

```diff
@@ -790,7 +790,7 @@ GEM
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    webpacker (3.3.1)
+    webpacker (4.0.0.rc.2)
```

Comment:
I'm afraid this is a necessary evil. It will fail with v3 because it uses a library that was deprecated and that breaks webpack 4.